### PR TITLE
Add_idefics3_model

### PR DESCRIPTION
**Pull request by Minestar6**
# Add Idefics3
添加了idefics3模型的PaddlePaddle 实现，包含模型代码、配置文件、单元测试。
# 对齐验证
## 精度对齐
前 20 个 token 的 logits 对齐结果如下：
* mean diff:  2.125230e-06 
* max diff:   2.622604e-05
token完全一致

### 训练数据
文本数据
* GSM8K SFT 数据
* 训练步数：300 steps
1_idefics3

多模态数据
### 训练结果

vl_idefics3

vl_idefics3_diff

### CODE REVIEW

**Comment by risemeup1111** [comment]
## Failed CI看板

| 流水线名称 | 问题标签 | 修复建议 | 日志片段 |
| ------------ | ---------- | ---------- | ---------- |
| CI_XPU / Clone-Linux / code-clone | 日志获取失败（Actions 日志 blob 404） | 重新运行 `Clone-Linux / code-clone`，并获取可用的完整日志后再判断代码根因 | [Job](https://github.com/PaddlePaddle/PaddleFormers/actions/runs/33249262527/job/99091977897) |

<details>
<summary>日志分析报告</summary>

**失败的测试 case**:
```text
Job: Clone-Linux / code-clone
Run: CI_XPU #33249262527, attempt 1
关键错误: 日志存储 blob 返回 404 The specified blob does not exist.
```

**根本原因分析**:

当前快照未取得 `code-clone` 的实际执行日志，仅确认 GitHub Actions 日志存储对象返回 404。因此无法验证是代码克隆检测失败、依赖问题还是 CI 基础设施问题，也不能将 PR 中新增的 Idefics3 代码认定为根因。

可确认的问题属于 CI 日志获取或日志存储可用性异常；失败 job 的状态本身不足以提供 PR 代码层面的根因证据。

**修复建议**:

1. 重新运行 `CI_XPU` 中的 `Clone-Linux / code-clone` job，并确认日志可下载。
2. 若重跑后仍返回 404，检查 GitHub Actions 日志保留或存储服务状态。
3. 获取完整日志后，再根据具体 clone 检测报错检查 PR 中新增文件的重复代码问题。

</details>

Powered by Nyanpasu with gpt-5.6-luna 默认推理级别, please check the suggestions carefully.
